pricing mode and extra consumption date (#57)
MewsSystems/gitbook-channel-manager-api: changelog.md

# Changelog

## 27th May 2020 12:00 UTC

> [Get Configuration] (https://mews-systems.gitbook.io/channel-manager-api/mews-api#get-configuration) extended. Now pricing mode, which shows whether a property is built in net or gross pricing environment, can be retrieved from Mews.

Reviewer: ```suggestion
[Get Configuration] (https://mews-systems.gitbook.io/channel-manager-api/mews-api#get-configuration) extended with pricing mode, which shows whether a property is built in net or gross pricing environment.
```
